assets/platforms/ruijie.yaml

```diff
@@ -0,0 +1,55 @@
+---
+platform-type: 'ruijie'
```

Comment:
same as the h3c comment!

Reply:
https://github.com/ktbyers/netmiko/blob/develop/netmiko/ruijie/ruijie_os.py

Reply:
lets go with `ruijie_rgos` I guess? that should stay consistent for scrapli and looks reasonable based on a quick search

Reply:
sure

Reply:
update as `ruijie_os`, I guess it's more suitable for network engineer views.